scripts/cleanup/region_endpoint_mapping.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# (C) Copyright IBM Corp. 2025.
+#!/usr/bin/env python -W ignore
+"""
+Retrieve Endpoint URL for the region pased in.
+"""
+
+def getPublicEndpointURL(region):
```

Comment:
This won't be stable.... just use a build pattern a string formatted with vpc_region_name or powervs_region_name.